_posts/2020-10-26-dynamically-changeable-logging.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "Dynamically changeable logging levels"
+date: 2020-10-26
+author: stanislav_knot
+---
+
+Logging is an indisputably important feature for applications.
+Logging monitors what the application is doing, but equally importantly,  it logs what the application is not doing. 
+Looking into application logs can help us prevent an application remaining or even getting into a faulty state.
+
+<!--more-->
+
+
+## Setting logging levels dynamically
+
+Strimzi components are split into the two groups regarding which implementation of `log4j` they use.
```

Comment:
Maybe add one sentence saying how there are many logging frameworks in Java and we use Log4j (plus link to Log4j website)?